src/properties.rs

```diff
@@ -30,6 +31,43 @@ where
         }
     }
 
+    /// Initialize display in column mode
+    pub fn init_column_mode(&mut self) -> Result<(), DI::Error> {
+        // TODO: Break up into nice bits so display modes can pick whathever they need
+        // Display is set up in column mode, i.e. a byte walks down a column of 8 pixels from
```

Comment:
Can you move this to the function docblock? Would be helpful to have in the rendered docs.

Reply:
I agree, that's why I was asking for naming ideas before. `DisplaySetup` would work, `DisplayConfiguration` also.